### PR TITLE
Adds a case study landing page

### PR DESCRIPTION
**Pull request by ConorOkus**
Waiting on a few fixes to the core theme and it to be upgraded on npm but should enable us to render a layout close to the following:

Screenshot 2023-05-15 at 19 05 04

### CODE REVIEW

**Comment by ConorOkus** [comment]
Currently getting the logos from the different projects in the right format...

File: docs/case-studies.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+      </div>
+
+  </template>
+
+  <template v-slot:all>
+  <div class="case-studies">
+    <div class="case-study-item">
+    <img src="./assets/blue-wallet.png" />
+    <h3><a href="https://bluewallet.io/" target="_blank">Blue Wallet</a></h3>
+    <p>A mobile bitcoin and lightning wallet.</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:201]:
```suggestion
    <p> A radically simple and powerful bitcoin and Lightning wallet
 </p>
```

File: docs/case-studies.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+   <div class="case-study-item">
+    <img src="./assets/bitkit.png" />
+    <h3><a href="https://porticoexchange.github.io/porticoexchangev2.github.io/" target="_blank">Bitkit</a></h3>
```
Comment (haleyberkoe) [inline docs/case-studies.md:205]:
```suggestion
    <h3><a href="https://bitkit.to/" target="_blank">Bitkit</a></h3>
```

File: docs/case-studies.md [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+    <p>A mobile wallet that features portable web profiles and passwordless web accounts</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:206]:
```suggestion
    <p> A mobile app that hands you the keys to your money, profile, contacts, and web accounts</p>
```

File: docs/case-studies.md [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+    <img src="./assets/kumuly.png" />
+    <h3><a href="https://twitter.com/kumulydev" target="_blank">Kumuly</a></h3>
+    <p>A Colomnia-based mobile bitcoin and Lightning wallet</p>
+    </div>
+    <div class="case-study-item">
+    <img src="./assets/lipa.png" />
+    <h3><a href="https://lipa.swiss/" target="_blank">Lipa</a></h3>
+    <p>Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:216]:
```suggestion
    <p>A Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
```

File: docs/case-studies.md [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/portico.png" />
+      <h3><a href="https://mercurywallet.com/" target="_blank">Portico</a></h3>
```
Comment (haleyberkoe) [inline docs/case-studies.md:220]:
```suggestion
      <h3><a href="https://porticoexchange.github.io/porticoexchangev2.github.io/" target="_blank">Portico</a></h3>
```

File: docs/case-studies.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+      <p>A DEX that enables people to swap between bitcoin layers and sidechains</p>
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/mercury.png" />
+      <h3><a href="https://porticoexchange.github.io/porticoexchangev2.github.io/" target="_blank">Mercury</a></h3>
```
Comment (haleyberkoe) [inline docs/case-studies.md:225]:
```suggestion
      <h3><a href="https://mercurywallet.com/" target="_blank">Mercury</a></h3>
```

File: docs/case-studies.md [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+      <p>A lighting wallet that enables BTC transfers without requiring an on-chain transaction</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:226]:
```suggestion
      <p>A layer 2 bitcoin wallet that enables users to send and swap bitcoin privately</p>
```

File: docs/case-studies.md [hunk 8]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/10101.png" />
+      <h3><a href="https://10101.finance/" target="_blank">10101</a></h3>
+      <p>An on-chain and off-chain wallet that allows trading</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:231]:
```suggestion
      <p>An on-chain and off-chain wallet infused with trading to unleash trustless decentralized finance</p>
```

File: docs/case-studies.md [hunk 9]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/mutiny.png" />
+      <h3><a href="https://mutinywallet.com/" target="_blank">Mutiny</a></h3>
+      <p>Mutiny is a web-first lightning wallet</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:236]:
```suggestion
      <p>A web-first unstoppable bitcoin wallet for everyone</p>
```

File: docs/case-studies.md [hunk 10]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/velas.png" />
+      <h3><a href="https://www.velascommerce.com/" target="_blank">Velas</a></h3>
+      <p>Enables businesses to integrate Lightning payments into websites, mobile applications, and more</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:241]:
```suggestion
      <p>Enables businesses to integrate Bitcoin and Lightning features into websites, mobile apps, and more</p>
```

File: docs/case-studies.md [hunk 11]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/atomic.png" />
+      <h3><a href="https://atomicdex.io/en/" target="_blank">AtomicDEX</a></h3>
+      <p>A multi-coin wallet, bridge, and DEX that can run on mobile and desktop</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:246]:
```suggestion
      <p>A multi-coin wallet, bridge, and DEX rolled into one app</p>
```

File: docs/case-studies.md [hunk 12]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/hydranet.png" />
+      <h3><a href="https://hydranet.ai/" target="_blank">Hydranet</a></h3>
+      <p>A DEX that offers trading between blockchains without the need for a layer 2 bridge</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:251]:
```suggestion
      <p>A layer 3 decentralized exchange - a solution for trading with native tokens between blockchains</p>
```

File: docs/case-studies.md [hunk 13]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+      <img src="./assets/cash-app-logo.png" />
+      <h3><a href="https://cash.app/" target="_blank">Cash App</a></h3>
+      <p>A mobile P2P payments app and bitcoin wallet</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:256]:
```suggestion
      <p>Send and spend, bank, and buy stocks or bitcoin</p>
```

File: docs/case-studies.md [hunk 14]
```diff
@@ -0,0 +1,317 @@
...
+    </div>
+    <div class="case-study-item">
+    <img src="./assets/teos.png" />
+    <h3><a href="https://github.com/talaia-labs/rust-teos" target="_blank">TEOS</a></h3>
+    <p>A bitcoin watchtower with a specific focus on lightning</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:261]:
```suggestion
    <p>A bitcoin watchtower with a specific focus on Lightning</p>
```

File: docs/case-studies.md [hunk 15]
```diff
@@ -0,0 +1,317 @@
...
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/vls.png" />
+    <h3><a href="https://vls.tech/" target="_blank">VLS</a></h3>
+    <p>Separates lightning private keys and security rule validation from nodes, into a discrete signing device</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:266]:
```suggestion
    <p>Separates Lightning private keys and security rule validation from nodes, into a discrete signing device</p>
```

File: docs/case-studies.md [hunk 16]
```diff
@@ -0,0 +1,317 @@
...
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/lexe.png" />
+    <h3><a href="https://github.com/lexe-tech" target="_blank">Lexe</a></h3>
+    <p>exe offers managed non-custodial Lightning nodes based on Intel SGX, accessible via mobile app</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:271]:
```suggestion
    <p>Managed non-custodial Lightning nodes based on Intel SGX, accessible via mobile app</p>
```

File: docs/case-studies.md [hunk 17]
```diff
@@ -0,0 +1,317 @@
...
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/voltage.png" />
+    <h3><a href="https://voltage.cloud/" target="_blank">Voltage</a></h3>
+    <p>Provides enterprise grade infrastructure for the Lightning Network</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:276]:
```suggestion
    <p>Enterprise-grade infrastructure for the Lightning Network</p>
```

File: docs/case-studies.md [hunk 18]
```diff
@@ -0,0 +1,317 @@
...
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/valera.png" />
+    <h3><a href="https://valera.co/" target="_blank">Valera</a></h3>
+    <p>Building a financial infrastructure suite for developers and users</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:281]:
```suggestion
    <p>A financial infrastructure suite for developers and users</p>
```

File: docs/case-studies.md [hunk 19]
```diff
@@ -0,0 +1,317 @@
...
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/kuutamo.png" />
+    <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">Kuutamo</a></h3>
+    <p>A turn-key, end-to-end solution for running self-hosted nodes, anywhere</p>
+  </div>
+  <div class="case-study-item">
+    <img src="./assets/c=.png" />
+    <h3><a href="https://cequals.xyz/" target="_blank">C=</a></h3>
+    <p>Building tools and services to help connect people to the Lightning Network</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:291]:
```suggestion
    <p>Tools and services that help connect people to the Lightning Network</p>
```

**Review by haleyberkoe — COMMENTED** [review]
Updated some links and edited copy to better reflect various projects/their websites.

File: docs/case-studies.md [hunk 20]
```diff
@@ -0,0 +1,334 @@
...
+    <div class="case-studies">
+     <div class="case-study-item">
+      <a href="https://10101.finance/" target="_blank"><img src="./assets/10101.png" /></a>
+      <h3><a href="https://10101.finance/" target="_blank">10101</a></h3>
+      <p>An on-chain and off-chain wallet infused with trading to unleash trustless decentralized finance</p>
+    </div>
+    <div class="case-study-item">
+      <a href="https://bitkit.to/" target="_blank"><img src="./assets/bitkit.png" /></a>
+      <h3><a href="https://bitkit.to/" target="_blank">Bitkit</a></h3>
+      <p>A mobile app that hands you the keys to your money, profile, contacts, and web accounts</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:43]:
```suggestion
      <p>The ultimate bitcoin toolkit, take charge of your digital life</p>
```

File: docs/case-studies.md [hunk 21]
```diff
@@ -0,0 +1,334 @@
...
+    </div>
+    <div class="case-study-item">
+      <a href="https://atomicdex.io/en/" target="_blank"><img src="./assets/atomic.png" /></a>
+      <h3><a href="https://atomicdex.io/en/" target="_blank">AtomicDEX</a></h3>
+      <p>A multi-coin wallet, bridge, and DEX rolled into one app</p>
+    </div>
+    <div class="case-study-item">
+      <a href="https://bitkit.to/" target="_blank"><img src="./assets/bitkit.png" /></a>
+      <h3><a href="https://bitkit.to/" target="_blank">Bitkit</a></h3>
+      <p>A mobile app that hands you the keys to your money, profile, contacts, and web accounts</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:223]:
```suggestion
      <p>The ultimate bitcoin toolkit, take charge of your digital life</p>
```

File: docs/case-studies.md [hunk 22]
```diff
@@ -0,0 +1,334 @@
...
+   Meet all the projects using LDK!
+</h1>
+
+<CodeSwitcher :languages="{all: 'All', mobile:'Mobile', web:'Web', desktop:'Desktop', custodial: 'Custodial', infra:'Infrastructure', library: 'Libraries', misc:'Misc',}">
+  <template v-slot:mobile>
+    <div class="case-studies">
+     <div class="case-study-item">
+      <a href="https://10101.finance/" target="_blank"><img src="./assets/10101.png" /></a>
+      <h3><a href="https://10101.finance/" target="_blank">10101</a></h3>
+      <p>An on-chain and off-chain wallet infused with trading to unleash trustless decentralized finance</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:38]:
```suggestion
      <p>An on and off-chain wallet infused with trading</p>
```

File: docs/case-studies.md [hunk 23]
```diff
@@ -0,0 +1,334 @@
...
+      </div>
+
+  </template>
+
+  <template v-slot:all>
+  <div class="case-studies">
+    <div class="case-study-item">
+      <a href="https://10101.finance/" target="_blank"><img src="./assets/10101.png" /></a>
+      <h3><a href="https://10101.finance/" target="_blank">10101</a></h3>
+      <p>An on-chain and off-chain wallet infused with trading to unleash trustless decentralized finance</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:213]:
```suggestion
      <p>An on and off-chain wallet infused with trading</p>
```

File: docs/case-studies.md [hunk 24]
```diff
@@ -0,0 +1,334 @@
...
+
+  </template>
+
+  <template v-slot:infra>
+
+  <div class="case-studies">
+  <div class="case-study-item">
+    <a href="https://cequals.xyz/" target="_blank"><img src="./assets/c=.png" /></a>
+    <h3><a href="https://cequals.xyz/" target="_blank">c=</a></h3>
+    <p>Tools and services that help connect people to the Lightning Network</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:134]:
```suggestion
    <p>Tools and services that connect people to the Lightning Network</p>
```

File: docs/case-studies.md [hunk 25]
```diff
@@ -0,0 +1,334 @@
...
+    </div>
+    <div class="case-study-item">
+    <a href="https://twitter.com/kumulydev" target="_blank"><img src="./assets/kumuly.png" /></a>
+    <h3><a href="https://twitter.com/kumulydev" target="_blank">Kumuly</a></h3>
+    <p>Colombian-based mobile bitcoin and Lightning wallet</p>
+    </div>
+    <div class="case-study-item">
+    <a href="https://lipa.swiss/" target="_blank"><img src="./assets/lipa.png" /></a>
+    <h3><a href="https://lipa.swiss/" target="_blank">Lipa</a></h3>
+    <p>A Swiss-based app that offers a bitcoin wallet for individuals and businesses</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:58]:
```suggestion
    <p>Swiss-based app that offers a bitcoin wallet for individuals and businesses</p>
```

File: docs/case-studies.md [hunk 26]
```diff
@@ -0,0 +1,334 @@
...
+   </div>
+   <div class="case-study-item">
+    <a href="https://github.com/lexe-tech" target="_blank"> <img src="./assets/lexe.png" /></a>
+    <h3><a href="https://github.com/lexe-tech" target="_blank">Lexe</a></h3>
+    <p>Managed non-custodial Lightning nodes based on Intel SGX, accessible via mobile app</p>
+  </div>
+  <div class="case-study-item">
+    <a href="https://lipa.swiss/" target="_blank"><img src="./assets/lipa.png" /></a>
+    <h3><a href="https://lipa.swiss/" target="_blank">Lipa</a></h3>
+    <p>A Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:268]:
```suggestion
    <p>Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
```

File: docs/case-studies.md [hunk 27]
```diff
@@ -0,0 +1,334 @@
...
+
+  </template>
+
+  <template v-slot:library>
+
+  <div class="case-studies">
+  <div class="case-study-item">
+    <a href="https://github.com/carlaKC/lndk" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/carlaKC/lndk" target="_blank">LNDK</a></h3>
+    <p>LNDK is an experimental attempt at using LDK to implement BOLT12 features for LND</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:166]:
```suggestion
    <p>An experimental attempt at using LDK to implement BOLT12 features for LND</p>
```

File: docs/case-studies.md [hunk 28]
```diff
@@ -0,0 +1,334 @@
...
+  </div>
+  <div class="case-study-item">
+    <a href="https://lipa.swiss/" target="_blank"><img src="./assets/lipa.png" /></a>
+    <h3><a href="https://lipa.swiss/" target="_blank">Lipa</a></h3>
+    <p>A Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
+  </div>
+  <div class="case-study-item">
+    <a href="https://github.com/carlaKC/lndk" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/carlaKC/lndk" target="_blank">LNDK</a></h3>
+    <p>LNDK is an experimental attempt at using LDK to implement BOLT12 features for LND</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:273]:
```suggestion
    <p>An experimental attempt at using LDK to implement BOLT12 features for LND</p>
```

File: docs/case-studies.md [hunk 29]
```diff
@@ -0,0 +1,334 @@
...
+   </div>
+   <div class="case-study-item">
+      <a href="https://github.com/kuutamolabs/lightning-knd" target="_blank"><img src="./assets/kuutamo.png" /></a>
+      <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">Kuutamo</a></h3>
+      <p>A turn-key, end-to-end solution for running self-hosted nodes, anywhere</p>
+   </div>
+   <div class="case-study-item">
+    <a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank">ldk-sample with Tor</a></h3>
+    <p>An experimentation with tor by adapting the ldk-sample node</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:258]:
```suggestion
    <p>An experimentation with tor that adapts the ldk-sample node</p>
```

File: docs/case-studies.md [hunk 30]
```diff
@@ -0,0 +1,334 @@
...
+      <div class="case-studies">
+         <div class="case-study-item">
+          <a href="https://github.com/BitcoinDevShop/hidden-lightning-network" target="_blank"><img src="./assets/github.png" /></a>
+          <h3><a href="https://github.com/BitcoinDevShop/hidden-lightning-network" target="_blank">Hidden Lightning Network</a></h3>
+          <p>Probes the Lightning Network for the detection of private channels</p>
+        </div>
+        <div class="case-study-item">
+          <a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank"><img src="./assets/github.png" /></a>
+          <h3><a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank">ldk-sample with Tor</a></h3>
+          <p>An experimentation with tor by adapting the ldk-sample node</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:192]:
```suggestion
          <p>An experimentation with tor that adapts the ldk-sample node</p>
```

File: docs/case-studies.md [hunk 31]
```diff
@@ -0,0 +1,334 @@
...
+      <div class="case-studies">
+      <div class="case-study-item">
+        <a href="https://atomicdex.io/en/" target="_blank"><img src="./assets/atomic.png" /></a>
+        <h3><a href="https://atomicdex.io/en/" target="_blank">AtomicDEX</a></h3>
+        <p>A multi-coin wallet, bridge, and DEX rolled into one app</p>
+      </div>
+      <div class="case-study-item">
+        <a href="https://hydranet.ai/" target="_blank"><img src="./assets/hydranet.png" /></a>
+        <h3><a href="https://hydranet.ai/" target="_blank">Hydranet</a></h3>
+        <p>A layer 3 decentralized exchange - a solution for trading with native tokens between blockchains</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:105]:
```suggestion
        <p>A layer 3 decentralized exchange, allowing trading with native tokens between blockchains</p>
```

File: docs/case-studies.md [hunk 32]
```diff
@@ -0,0 +1,334 @@
...
+    </div>
+    <div class="case-study-item">
+      <a href="https://bluewallet.io/" target="_blank"><img src="./assets/blue-wallet.png" /></a>
+      <h3><a href="https://bluewallet.io/" target="_blank">Blue Wallet</a></h3>
+      <p>A radically simple and powerful bitcoin and Lightning wallet</p>
+    </div>
+    <div class="case-study-item">
+      <a href="https://cequals.xyz/" target="_blank"><img src="./assets/c=.png" /></a>
+      <h3><a href="https://cequals.xyz/" target="_blank">c=</a></h3>
+      <p>Tools and services that help connect people to the Lightning Network</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:233]:
```suggestion
      <p>Tools and services that connect people to the Lightning Network</p>
```

File: docs/case-studies.md [hunk 33]
```diff
@@ -0,0 +1,334 @@
...
+    </div>
+    <div class="case-study-item">
+      <a href="https://cash.app/" target="_blank"> <img src="./assets/cash-app-logo.png" /></a>
+      <h3><a href="https://cash.app/" target="_blank">Cash App</a></h3>
+      <p>Send and spend, bank, and buy stocks or bitcoin</p>
+    </div>
+    <div class="case-study-item">
+      <a href="https://hydranet.ai/" target="_blank"><img src="./assets/hydranet.png" /></a>
+      <h3><a href="https://hydranet.ai/" target="_blank">Hydranet</a></h3>
+      <p>A layer 3 decentralized exchange - a solution for trading with native tokens between blockchains</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:243]:
```suggestion
      <p>A layer 3 decentralized exchange, allowing trading with native tokens between blockchains</p>
```

File: docs/case-studies.md [hunk 34]
```diff
@@ -0,0 +1,321 @@
...
+  </div>
+  <div class="case-study-item">
+    <a href="https://github.com/kuutamolabs/lightning-knd" target="_blank"><img src="./assets/kuutamo.png" /></a>
+    <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">Kuutamo</a></h3>
+    <p>A turn-key, end-to-end solution for running self-hosted nodes, anywhere</p>
+  </div>
+  <div class="case-study-item">
+    <a href="https://github.com/carlaKC/lndk" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/carlaKC/lndk" target="_blank">LNDK</a></h3>
+    <p>LNDK is an experimental attempt at using LDK to implement BOLT12 features for LND</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:138]:
```suggestion
    <p>A standalone daemon that aims to externally implement bolt 12 functionality for LND</p>
```

File: docs/case-studies.md [hunk 35]
```diff
@@ -0,0 +1,321 @@
...
+  </div>
+  <div class="case-study-item">
+    <a href="https://lipa.swiss/" target="_blank"><img src="./assets/lipa.png" /></a>
+    <h3><a href="https://lipa.swiss/" target="_blank">Lipa</a></h3>
+    <p>A Swiss-based mobile app that offers a bitcoin wallet for individuals and businesses</p>
+  </div>
+  <div class="case-study-item">
+    <a href="https://github.com/carlaKC/lndk" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/carlaKC/lndk" target="_blank">LNDK</a></h3>
+    <p>LNDK is an experimental attempt at using LDK to implement BOLT12 features for LND</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:260]:
```suggestion
    <p>A standalone daemon that aims to externally implement bolt 12 functionality for LND</p>
```

**Review by haleyberkoe — COMMENTED** [review]
Updated LNDK copy

File: docs/case-studies.md [hunk 36]
```diff
@@ -0,0 +1,321 @@
...
+   </div>
+   <div class="case-study-item">
+    <a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank"><img src="./assets/github.png" /></a>
+    <h3><a href="https://github.com/TonyGiorgio/ldk-sample-tor" target="_blank">ldk-sample with Tor</a></h3>
+    <p>An experimentation with tor by adapting the ldk-sample node</p>
+   </div>
+   <div class="case-study-item">
+    <a href="https://github.com/lexe-tech" target="_blank"> <img src="./assets/lexe.png" /></a>
+    <h3><a href="https://github.com/lexe-tech" target="_blank">Lexe</a></h3>
+    <p>Managed non-custodial Lightning nodes based on Intel SGX, accessible via mobile app</p>
```
Comment (haleyberkoe) [inline docs/case-studies.md:250]:
```suggestion
    <p>Managed non-custodial Lightning nodes inside secure hardware</p>
```

**Review by haleyberkoe — COMMENTED** [review]
Updated Lexe copy

File: docs/case-studies.md [hunk 37]
```diff
@@ -0,0 +1,321 @@
...
+      <p>A layer 3 decentralized exchange - a solution for trading with native tokens between blockchains</p>
+    </div>
+   <div class="case-study-item">
+    <a href="https://twitter.com/kumulydev" target="_blank"><img src="./assets/kumuly.png" /></a>
+    <h3><a href="https://twitter.com/kumulydev" target="_blank">Kumuly</a></h3>
+    <p>Colombian-based mobile bitcoin and Lightning wallet</p>
+   </div>
+   <div class="case-study-item">
+      <a href="https://github.com/kuutamolabs/lightning-knd" target="_blank"><img src="./assets/kuutamo.png" /></a>
+      <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">Kuutamo</a></h3>
```
Comment (haleyberkoe) [inline docs/case-studies.md:239]:
```suggestion
      <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">kuutamo</a></h3>
```

File: docs/case-studies.md [hunk 38]
```diff
@@ -0,0 +1,321 @@
...
+
+  <div class="case-studies">
+  <div class="case-study-item">
+    <a href="https://cequals.xyz/" target="_blank"><img src="./assets/c=.png" /></a>
+    <h3><a href="https://cequals.xyz/" target="_blank">c=</a></h3>
+    <p>Tools and services that help connect people to the Lightning Network</p>
+  </div>
+  <div class="case-study-item">
+    <a href="https://github.com/kuutamolabs/lightning-knd" target="_blank"><img src="./assets/kuutamo.png" /></a>
+    <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">Kuutamo</a></h3>
```
Comment (haleyberkoe) [inline docs/case-studies.md:132]:
```suggestion
    <h3><a href="https://github.com/kuutamolabs/lightning-knd" target="_blank">kuutamo</a></h3>
```

**Pull request merged**